testcases/misc_tests/p11sak_test.sh

```diff
@@ -124,303 +134,400 @@ echo "** Now checking output files to determine PASS/FAIL of tests - 'p11sak_tes
 grep -q 'p11sak-des' $P11SAK_DES_PRE
 rc=$?
 if [ $rc = 0 ]; then
-echo "* TESTCASE generate-key des PASS Generated random DES key"
+echo "* TESTCASE - [PASS] -- generate-key des Generated random DES key"
```

Comment:
This does not work with the CI parser for test case results.  Format is/has to be "TESTCASE <casename> [PASS | FAIL] <description>".  This applies to all cases.

Reply:
Seems like HTML tricked me here.  Format should be `TESTCASE <testcasename> [PASS|FAIL|SKIP] <description>`.  Hopefully setting it in code will keep the tags.

Reply:
Done